pkg/allocation/go/allocation.swagger.json

```diff
@@ -4,6 +4,11 @@
     "title": "proto/allocation/allocation.proto",
     "version": "version not set"
   },
+  "tags": [
+    {
+      "name": "AllocationService"
+    }
+  ],
```

Comment:
This tags is generated by `protoc-gen-openapiv2`. This is always printed and cannot be disabled.